marquez_airflow/extractors/bigquery_extractor.py

```diff
@@ -56,12 +56,13 @@ def extract(self) -> [StepMetadata]:
             }
         )]
 
-    def extract_on_complete(self) -> [StepMetadata]:
+    def extract_on_complete(self, task_instance) -> [StepMetadata]:
         steps_meta = self.extract()
 
         client = bigquery.Client()
 
-        job_name = get_job_name(task=self.operator)
+        # job_name = get_job_name(task=self.operator)
```

Comment:
Safe to remove the commented out line here?

Reply:
done.